tests/ProjectCodeTest.php

```diff
@@ -66,14 +67,14 @@ public function testHasNullOr($method)
      */
     public function testHasAll($method)
     {
-        $fullMethodName = 'all' . ucfirst($method);
+        $fullMethodName = 'all'.ucfirst($method);
 
-        $correct = strpos((string) self::$assertDocComment,'@method static void ' . $fullMethodName);
+        $correct = strpos((string) self::$assertDocComment, '@method static void '.$fullMethodName);
         if (!$correct) {
             $correct = in_array($fullMethodName, self::$mixinMethodNames, true);
         }
 
-        if ($correct === false) {
+        if (false === $correct) {
```

Comment:
Free to ignore: not really a fan of yoda conditionals, i'd prefer to skip that rule.

Reply:
I can see their value, but Im not in love with them. 

This is reverted.